packages/dbos-cloud/cloudutils.ts

```diff
@@ -1,10 +1,41 @@
 import { DBOSCloudCredentials, dbosEnvPath } from "./login";
+import TransportStream = require("winston-transport");
 import fs from "fs";
 import { spawn, StdioOptions } from 'child_process';
-import { GlobalLogger } from "../../src/telemetry/logs";
+import { transports, createLogger, format, Logger } from "winston";
+
+export function getLogger(): Logger {
+  const winstonTransports: TransportStream[] = [];
+  winstonTransports.push(
+    new transports.Console({
+      format: consoleFormat,
+      level:  "info",
+    })
+  );
+  return createLogger({ transports: winstonTransports });
+}
+
+const consoleFormat = format.combine(
+  format.errors({ stack: true }),
+  format.timestamp(),
+  format.colorize(),
+  format.printf((info) => {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    const { timestamp, level, message, stack } = info;
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-assignment
+    const ts = timestamp.slice(0, 19).replace("T", " ");
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-assignment
+    const formattedStack = stack?.split("\n").slice(1).join("\n");
+
+    const messageString: string = typeof message === "string" ? message : JSON.stringify(message);
+
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+    return `${ts} [${level}]: ${messageString} ${stack ? "\n" + formattedStack : ""}`;
+  })
```

Comment:
I don't think we need to use this. Likely we can just use the default formatting.

Reply:
Next PR?